demo/index.html

```diff
@@ -41,38 +46,82 @@
         <span class="mt-2" style="font-size: 20px"
           >a library for generating high-quality, printable maps.</span
         >
       </h1>
 
-      <div class="mt-5 example-01">
+      <div class="mt-5" id="example-01">
         <p>
           Give <strong>inkmap</strong> a JSON spec and it will use it to
           generate a PNG image in the background.
         </p>
-        <div class="row align-items-center">
-          <print-spec class="col-8" editable="true"></print-spec>
-          <custom-button class="col-3">
-            Generate!
-          </custom-button>
+        <ul class="nav nav-tabs">
+          <li class="nav-item">
+            <a
+              class="nav-link active"
+              data-toggle="tab"
+              href=".example-01-result"
+              >Result</a
+            >
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" data-toggle="tab" href=".example-01-source"
+              >Source</a
+            >
+          </li>
+        </ul>
+        <div class="tab-content mt-2">
+          <div class="tab-pane example-01-result show active">
+            <div class="row align-items-center">
+              <print-spec class="col-8" editable="true"></print-spec>
+              <custom-button class="col-3"> Generate! </custom-button>
+            </div>
+          </div>
+          <div class="tab-pane example-01-source">
+            <pre
+              class="rounded small"
+            ><code class="js"><%- example01 %></code></pre>
+          </div>
         </div>
       </div>
 
-      <div class="mt-5 example-02">
+      <div class="mt-5" id="example-02">
         <h2>Showing progress</h2>
         <p>
           <strong>inkmap</strong> also enables finer monitoring by making the
           jobs status observable.
         </p>
-        <div class="row align-items-center">
-          <print-spec class="col-8" editable="false"></print-spec>
-          <custom-button class="col-3">
-            Generate!
-          </custom-button>
+        <ul class="nav nav-tabs">
+          <li class="nav-item">
+            <a
+              class="nav-link active"
+              data-toggle="tab"
+              href=".example-02-result"
+              >Result</a
+            >
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" data-toggle="tab" href=".example-02-source"
+              >Source</a
+            >
+          </li>
+        </ul>
+        <div class="tab-content mt-2">
+          <div class="tab-pane example-02-result show active">
+            <div class="row align-items-center">
+              <print-spec class="col-8" editable="false"></print-spec>
+              <custom-button class="col-3"> Generate! </custom-button>
+            </div>
+          </div>
+          <div class="tab-pane example-02-source">
+            <pre
+              class="rounded small"
+            ><code class="js"><%- example02 %></code></pre>
+          </div>
```

Comment:
remove `<script src="app.js"></script>` cause it's managed by `WebpackHTMLPlugin`